arelle/ModelObjectFactory.py

```diff
@@ -45,9 +45,11 @@ def parser(
         modelXbrl: ModelXbrl,
         baseUrl: str | None,
         target: None = None,
+        no_network: bool = True
 ) -> tuple[etree.XMLParser, KnownNamespacesModelObjectClassLookup, DiscoveringClassLookup]:
     moduleObject_init()  # init ModelObject globals
-    _parser = etree.XMLParser(recover=True, huge_tree=True, target=target, resolve_entities=False)
+    _parser = etree.XMLParser(recover=True, huge_tree=True, target=target,
+                              resolve_entities=False, no_network=no_network)
```

Comment:
`no_network=True` is the library default. Is there a case where we want this to be `False`?